core/services/chainlink/application.go

```diff
@@ -148,16 +148,23 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 	scryptParams := utils.GetScryptParams(config)
 	keyStore := keystore.New(store.DB, scryptParams)
 
+	telemetryIngressClient := synchronization.TelemetryIngressClient(&synchronization.NoopTelemetryIngressClient{})
 	explorerClient := synchronization.ExplorerClient(&synchronization.NoopExplorerClient{})
 	statsPusher := synchronization.StatsPusher(&synchronization.NoopStatsPusher{})
-	monitoringEndpoint := ocrtypes.MonitoringEndpoint(&telemetry.NoopAgent{})
+	monitoringEndpointGen := telemetry.MonitoringEndpointGenerator(&telemetry.NoopAgent{})
 
 	if config.ExplorerURL() != nil {
 		explorerClient = synchronization.NewExplorerClient(config.ExplorerURL(), config.ExplorerAccessKey(), config.ExplorerSecret(), config.StatsPusherLogging())
 		statsPusher = synchronization.NewStatsPusher(store.DB, explorerClient)
-		monitoringEndpoint = telemetry.NewAgent(explorerClient)
+		monitoringEndpointGen = telemetry.NewExplorerAgent(explorerClient)
```

Comment:
I believe these two things may need to coexist?

Reply:
`monitoringEndpintGen` is intended to be a replacement for `monitoringEndpoint` (my other comment has more context)